assets/sass/components/tour-tooltip/_googlesitekit-tour-tooltip.scss

```diff
@@ -60,9 +59,8 @@
 	font-family: $f-secondary;
 	font-size: $fs-body-lg;
 	font-weight: $fw-medium;
-	letter-spacing: $ls-m;
-	line-height: $lh-body-lg;
-	margin-bottom: 4px;
+	line-height: $lh-display-lg;
```

Comment:
Why change the line height variable here? This changes it from `1.33` to `1.1`. The design uses `12/16` for the `size/height` so `1.33` is the closer/proper value to use. This seems to be why the VRT images changed, which probably shouldn't need to change as part of this issue.

Reply:
The original line height cause feature tour steps with multi-line headers look ugly. I think when we created this design for feature tours, nobody thought about multi-line headers and didn't account for it. I can revert it, but it will look worse.

Reply:
Ok, reverted.